docs/en_US/ResearchBlog/NASComparison.md

```diff
@@ -0,0 +1,81 @@
+# NAS Algorithms Comparison
+Train and Compare NAS models including Autokeras, DARTS, ENAS and NAO.
+
+Their source code link is as below:
+
+- Autokeras: [https://github.com/jhfjhfj1/autokeras](https://github.com/jhfjhfj1/autokeras)
+
+- DARTS: [https://github.com/quark0/darts](https://github.com/quark0/darts)
+
+- ENAS: [https://github.com/melodyguan/enas](https://github.com/melodyguan/enas)
+
+- NAO: [https://github.com/renqianluo/NAO](https://github.com/renqianluo/NAO)
+
+## Experiment Description
+
+To avoid over-fitting in **CIFAR-10** (I hate this dataset) , we also compare the models in the other five datasets including Fashion-MNIST, CIFAR-100, OUI-Adience-Age, ImageNet-10-1 (subset of ImageNet), ImageNet-10-2 (another subset of ImageNet). We just sample a subset with 10 different labels from ImageNet to make ImageNet-10-1 or ImageNet-10-2.
+
+
+
+| Dataset                                                      | Training Size | Numer of Classes | Descriptions                                                 |
+| :----------------------------------------------------------- | ------------- | ---------------- | ------------------------------------------------------------ |
+| [Fashion-MNIST](<https://github.com/zalandoresearch/fashion-mnist>) | 60,000        | 10               | T-shirt/top, trouser, pullover, dress, coat, sandal, shirt, sneaker, bag and ankle boot. |
+| [CIFAR-10](<https://www.cs.toronto.edu/~kriz/cifar.html>)    | 50,000        | 10               | Airplanes, cars, birds, cats, deer, dogs, frogs, horses, ships and trucks. |
+| [CIFAR-100](<https://www.cs.toronto.edu/~kriz/cifar.html>)   | 50,000        | 100              | Similar to CIFAR-10 but with 100 classes and 600 images each. |
+| [OUI-Adience-Age](<https://talhassner.github.io/home/projects/Adience/Adience-data.html>) | 26,580        | 8                | 8 age groups/labels (0-2, 4-6, 8-13, 15-20, 25-32, 38-43, 48-53, 60-). |
+| [ImageNet-10-1](<http://www.image-net.org/>)                 | 9,750         | 10               | Coffee mug, computer keyboard, dining table, wardrobe, lawn mower, microphone, swing, sewing machine, odometer and gas pump. |
+| [ImageNet-10-2](<http://www.image-net.org/>)                 | 9,750         | 10               | Drum, banj, whistle, grand piano, violin, organ, acoustic guitar, trombone, flute and sax. |
+
+
+
+We do not change the default fine-tuning technique in their source code. In order to match each task, the codes of input image shape and output numbers are changed.  
+
+Search phase time for all NAS methods is **two days** as well as the retrain time.  Average results are reported based on **three repeat times**.
```

Comment:
talking about training time, do you need to provide hardware info as reference?

Reply:
Thank you. We need to provide hardware info as reference. I will do it soon.